packages/lesswrong/lib/collections/users/fragments.ts

```diff
@@ -170,9 +172,8 @@ registerFragment(`
     abTestKey
     abTestOverrides
 
-    sortDrafts
+    sortDraftsBy
```

Comment:
Not sure why this is changing, assuming it's changing concurrently with whatever's using the result of the query